src/shorthands/borderColor.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import directionalProperty from '../helpers/directionalProperty'
+
+/**
+ * The border-color shorthand accepts up to four values, including null to skip a value, and uses the directional-property mixin to map them to their respective directions.
+ * @example
+ * // Styles as object usage
+ * const styles = {
+ *   ...borderColor('red', 'green', 'blue', 'yellow')
+ * }
+ *
+ * // styled-components usage
+ * const div = styled.div`
+ *   ${borderColor('red', 'green', 'blue', 'yellow')}
+ * `
+ *
+ * // CSS as JS Output
+ *
+ * div {
+ *   'border-top-color': 'red',
+ *   'border-right-color': 'green',
+ *   'border-bottom-color': 'blue',
+ *   'border-left-color': 'yellow'
+ * }
+ */
+
+function borderColor(...values: Array<string|null>) {
```

Comment:
If you set `null` as a type flow will think that's an `any`, which we need to avoid. Instead of saying that, do `param: ?string`. That'll tell flow that the param is null-able!